File: Data Structures & Algorithms/arranging-coins/submission-0.cpp
class Solution {
public:
    int arrangeCoins(int n) {
       int sum=n;
       int count=-1;
       for(int i=1;i<n;i++){
        sum=sum-i;
        count++;
        if(sum<0){
            return count;
        }
       } 
    }
};

File: Data Structures & Algorithms/number-of-sub-arrays-of-size-k-and-average-greater-than-or-equal-to-threshold/submission-0.java
class Solution {
    public int numOfSubarrays(int[] arr, int k, int threshold) {
        int n=arr.length;
        int left=0;
        int sum=0;
        int count=0;

        int target=k*threshold;
        for(int i=0;i<n;i++){
            sum=sum+arr[i];

            if(i-left+1==k){
                if (sum >= target) {
                    count++;
                }
                sum -= arr[left]; 
                left++;
            }
        }
        return count;
    }
}